modules/network-observability-ebpf-manager-operator.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-ebpf-manager-operator_{context}"]
+= Working with the eBPF Manager Operator
+
+The eBPF Manager Operator reduces the attack surface and ensures compliance, security, and conflict prevention by managing all eBPF programs. Network observability can use eBPF Manager Operator to load hooks. This eliminates the need to provide the eBPF Agent with privileged mode or additional Linux capabilities like `CAP_BPF` and `CAP_PERFMON`. The eBPF Manager Operator with network observability is only supported on 64-bit AMD architecture.
+
+:FeatureName: eBPF Manager Operator with network observability
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Operator Hub*.
+. Install *eBPF Manager*.
+. Check *Workloads* -> *Pods* in the `bpfman` namespace to make sure they are all up and running.
+. Configure the `FlowCollector` custom resource to use the eBPF Manager Operator:
++
+.Example `FlowCollector` configuration
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  agent:
+    ebpf:
+      features:
+        - EbpfManager
+----
+
+.Verification
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Click *eBPF Manager Operator* -> *All instances* tab.
++
+There should be a `BpfApplication` named `netobserv` and a pair of `BpfProgram` objects, one for Traffic Control (TCx) ingress and another for TCx egress, for each node. There might be more if you enable other eBPF Agent features.
```

Comment:
```suggestion
For each node, verify that a `BpfApplication` named `netobserv` and a pair of `BpfProgram` objects, one for Traffic Control (TCx) ingress and another for TCx egress, exist. If you enable other eBPF Agent features, you might have more objects.
```
- IBM Style advises against starting a sentence with "There...". To make this paragraph clearer, add a noun after `BpfApplication` and be more specific about what there might be more of if you enable other eBPF agent features.